includes/templates/single-amp-story.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Template for amp_story post type.
+ *
+ * @package AMP
+ */
+
+the_post();
+
+?>
+<!DOCTYPE html>
+<html amp <?php language_attributes(); ?>>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+
+	<?php
+	/**
+	 * Prints scripts or data in the head tag on the front end.
+	 *
+	 * @since 1.3
+	 */
+	do_action( 'amp_story_head' );
+	?>
+</head>
+<body>
+<?php
+$metadata = amp_get_schemaorg_metadata();
+if ( isset( $metadata['publisher']['logo']['url'] ) ) {
+	$publisher_logo_src = $metadata['publisher']['logo']['url'];
+} elseif ( isset( $metadata['publisher']['logo'] ) && is_string( $metadata['publisher']['logo'] ) ) {
+	$publisher_logo_src = $metadata['publisher']['logo'];
+} else {
+	$publisher_logo_src = admin_url( 'images/wordpress-logo.png' );
+}
+$publisher = isset( $metadata['publisher']['name'] ) ? $metadata['publisher']['name'] : get_option( 'blogname' );
+
+$meta_images = AMP_Story_Media::get_story_meta_images();
+?>
+<amp-story
```

Comment:
`amp-story` would ideally also be added to the `post_content` for portability, we could do that on the PHP side initially for simplicity, e.g. via filtering while saving.